tests/epyccel/test_epyccel_mod.py

```diff
@@ -11,7 +11,7 @@ def modulo_i_i(x, y):
         return x % y
 
     f = epyccel(modulo_i_i, language=language)
-    x = randint(1e6)
+    x = randint(-1e6, 1e6)
     y = randint(low=1, high=100) # low=1 for avoid zero-division error
```

Comment:
Could we also add a test with a negative `y` please. Beware, `y` must never be `0`!